cli/tests/activate.bats

```diff
@@ -2770,14 +2782,14 @@ attach_runs_hooks_once() {
 
   "$FLOX_BIN" activate -- bash -c "echo > activate_finished && echo > \"$TEARDOWN_FIFO\"" 2> output &
 
-  cat activate_finished
+  timeout 2s cat activate_finished || echo "activation didn't complete in time" >&2
```

Comment:
blocking question: will `|| echo` cause the test to pass? Do we need `&& false` or `&& return 1` as well?